enterprise/backend/src/metabase_enterprise/transforms/interface.clj

```diff
@@ -4,15 +4,6 @@
   [transform]
   (-> transform :source :type keyword))
 
-(defmulti source-db-id
```

Comment:
This was not currently being used, and in fact we want to *avoid* writing code that assumes transforms have a single source db. We would like to keep the option open for python transforms to do cross-db joins in future, since they're zero cost

Reply:
makes sense